compiler/luci-interpreter/src/kernels/FullyConnected.cpp

```diff
@@ -97,12 +104,23 @@ void FullyConnected::configure()
 void FullyConnected::execute() const
 {
   const bool is_hybrid =
-    (input()->element_type() == DataType::FLOAT32 && weights()->element_type() == DataType::S4 &&
+    (input()->element_type() == DataType::FLOAT32 &&
+     (weights()->element_type() == DataType::S4 || weights()->element_type() == DataType::U4) &&
      output()->element_type() == DataType::FLOAT32 &&
      (!bias() || bias()->element_type() == DataType::FLOAT32));
   if (is_hybrid)
   {
-    evalHybridWI4AF32();
+    switch (weights()->element_type())
+    {
+      case DataType::S4:
+        evalHybridWI4AF32();
+        break;
+      case DataType::U4:
+        evalHybridWU4AF32();
+        break;
+      default:
+        throw std::runtime_error("luci-intp FullyConnected(2) Unsupported type.");
+    }
```

Comment:
maybe add full name - `luci-interpreter`?

Reply:
Apologizes as I changed all other ops to have `luci-intp` as shorter prefix for unsupported type message. Before this, it was hard to know which Op that dropped `Unsupported type`.

Reply:
These is already `FullyConnected(2)` so, looks like `FullyConnected(3)` would be better to find where this runtime error happened.